File: 백준/Bronze/15873. 공백 없는 A＋B/공백 없는 A＋B.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int num = scanner.nextInt();

        if(num % 10 == 0) {
            num /= 100;
            System.out.println(10+num);
        }
        else {
            int a = num%10;
            num /= 10;
            if (num == 10)
                System.out.println(10 + a);
            else
                System.out.println(a + num);

        }
    }
}

File: 백준/Bronze/15873. 공백 없는 A＋B/공백 없는 A＋B.py
num = int(input())

if(num % 10 == 0):
    a = num // 100
    print(10 + a)

else :
    b = num % 10
    a = num // 10
    
    if(a == 10):
        print(10 + b)

    else :
        print(a + b)